--- lib/BUILD.bazel
+++ dist/lib.BUILD.bazel
@@ -1,13 +1,6 @@
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
-load(":platforms_test.bzl", "platforms_test_suite")
+"Public BUILD file for lib package"
 
-exports_files(
-    ["BUILD.bazel"] + glob(["*.bzl"]),
-    visibility = [
-        "//dist:__pkg__",
-        "//docs:__pkg__",
-    ],
-)
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
 bzl_library(
     name = "platforms",
@@ -15,5 +8,3 @@
     visibility = ["//visibility:public"],
     deps = ["@local_config_platform//:constraints.bzl"],
 )
-
-platforms_test_suite(name = "platforms_test")
