--- rules/BUILD.bazel
+++ dist/rules.BUILD.bazel
@@ -1,37 +1,5 @@
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+"Public BUILD file for rules package"
 
 package(default_visibility = ["//visibility:public"])
 
 exports_files(glob(["*.bzl"]))
-
-bzl_library(
-    name = "diff",
-    srcs = ["diff.bzl"],
-    visibility = ["//visibility:public"],
-    deps = ["@aspect_bazel_lib//lib:write_source_files"],
-)
-
-bzl_library(
-    name = "github_repo",
-    srcs = ["github_repo.bzl"],
-    visibility = ["//visibility:public"],
-    deps = [
-        ":bazel_tools_repo",
-        ":http",
-    ],
-)
-
-bzl_library(
-    name = "http",
-    srcs = ["http.bzl"],
-    visibility = ["//visibility:public"],
-)
-
-bzl_library(
-    name = "bazel_tools_repo",
-    srcs = [
-        "@bazel_tools//tools/build_defs/repo:cache.bzl",
-        "@bazel_tools//tools/build_defs/repo:http.bzl",
-        "@bazel_tools//tools/build_defs/repo:utils.bzl",
-    ],
-)
