--- sh/BUILD.bazel
+++ dist/BUILD.sh.bazel
@@ -1,5 +1,3 @@
-load("@aspect_bazel_lib//lib:bats.bzl", "bats_test")
-
 package(
     default_visibility = ["//visibility:public"],
 )
@@ -18,15 +16,3 @@
     },
     tags = ["manual"],
 )
-
-bats_test(
-    name = "shasums_test",
-    srcs = ["shasums.bats"],
-    data = ["shasums.sh"],
-)
-
-bats_test(
-    name = "tag_test",
-    srcs = ["tag.bats"],
-    data = ["tag.sh"],
-)
